src/components/common/form/message-item/MessageItem.spec.jsx

```diff
@@ -1,20 +1,19 @@
 import { screen } from '@testing-library/react';
-import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
 import MessageItem from './MessageItem';
+import { renderWithApplicationContext } from '../../../../test/helpers/TestApplicationContext';
 
 describe('MessageItem', () => {
   window.HTMLElement.prototype.scrollIntoView = jest.fn();
   it('should render a summary component with a single date and messages details', () => {
     const summaryMessages = {
-      update: {
-        template: `datesMoved`,
-        variables: { startDate: '2022-09-21' },
-      },
+      key: 'datesMoved',
```

Comment:
This key is in the tests but I can't see where it's used in the component, is it needed?